### DISCUSSION TITLE
无法安装 exec、shell_exec 函数未能通过

### DISCUSSION THREAD

**Discussion by Nick838** [lsky-org/lsky-pro#797] [问答]
准备安装时，程序检测 exec、shell_exec 函数未能通过，无法安装

问过虚拟主机客服，答：您好，抱歉, 虚拟主机是共享资源,环境, 要考虑整个服务器安全问题,目前禁用了///

是否就这样放弃..!?

![lsky111](https://github.com/lsky-org/lsky-pro/assets/19961436/eb8606fb-f9ab-4d44-b120-2b74ab35502d)

**Comment by akatsukiro** [comment]
正是因为需要这几个函数支持，所以可以说几乎所有虚拟主机都无法安装 Lsky 。

**Reply by Nick838** [reply]
明白了!...
